### ISSUE TITLE
Epic: Javadocs Accessibility Testing

### ISSUE THREAD

**Issue by natalie-bernhard** [OpenLiberty/openliberty.io#3647]
This will be the epic used to keep track of the issues regarding Javadocs Accessibility Testing.  Automated tests should be written using the Cypress package for the IBM Equal Access Checker (https://www.npmjs.com/package/cypress-accessibility-checker).  More info can be found here: https://www.ibm.com/able/toolkit/tools/#develop.  The following issues are a part of this epic:

- #3648 
- #3649 
- #3650 
- #3651 
- #3652 
- #3653
